[2024-04-06 20:56:39,194] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2024-04-05T00:00:00+00:00 [queued]>
[2024-04-06 20:56:39,216] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2024-04-05T00:00:00+00:00 [queued]>
[2024-04-06 20:56:39,217] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-04-06 20:56:39,220] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-04-06 20:56:39,224] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-04-06 20:56:39,271] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2024-04-05 00:00:00+00:00
[2024-04-06 20:56:39,287] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2024-04-05T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/file_upload_gcs.py', '--cfg-path', '/tmp/tmp4vaou2zs', '--error-file', '/tmp/tmp1bu_uyhr']
[2024-04-06 20:56:39,282] {standard_task_runner.py:52} INFO - Started process 9982 to run task
[2024-04-06 20:56:39,292] {standard_task_runner.py:77} INFO - Job 3: Subtask local_to_gcs_task
[2024-04-06 20:56:39,369] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2024-04-05T00:00:00+00:00 [running]> on host dd74f47d4afe
[2024-04-06 20:56:39,467] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2024-04-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-05T00:00:00+00:00
[2024-04-06 20:56:56,648] {python.py:175} INFO - Done. Returned value was: None
[2024-04-06 20:56:56,727] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20240405T000000, start_date=20240406T205639, end_date=20240406T205656
[2024-04-06 20:56:56,905] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-04-06 20:56:57,242] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
